nipype/interfaces/setup.py

```diff
@@ -24,6 +24,7 @@ def configuration(parent_package='', top_path=None):
     config.add_subpackage('mne')
     config.add_subpackage('mrtrix')
     config.add_subpackage('mrtrix3')
+    config.add_subpackage('niftyseg')
```

Comment:
Yes sounds good. I jut did the changes and I renamed the function in niftyreg base for something more general like no_nifty_package.

What do you think about the changes? 

Also do you prefer to import NiftyRegCommand like this:

import ..niftyreg.base.NiftyRegCommand as NiftySegCommand 

Or do as I did?

I will do then the edits on niftyfit when you agree with those changes.

Kind Regards,
Ben